bin/anthology/utils.py

```diff
@@ -414,7 +416,7 @@ def parse_element(xml_element):
             attrib["pdf"] = (
                 element.text
```

Comment:
What was the reason again we don't use `infer_url(element.text, template=data.PDF_LOCATION_TEMPLATE)` here? It seems logically equivalent.

Reply:
because infer_url creates a canonical URL for pages, not for PDFs.

For example, if you host a mirror, `infer_url("2020.acl-main.1")` should return `https://aclweb.org/anthology/2020.acl-main.1` because that is the canonical URL, but PDFs should link to the mirror and not the primary site.

Reply:
But isn't that what the `template=` argument is for?

Reply:
Possibly.  It is the same with attachments, but I did not write that code and would have to look at it more closely.  Maybe add an issue to refactor that?